notifier/orb.yaml

```diff
@@ -1,12 +1,56 @@
 version: 2.1
 description: "Tools for notifying external services about CI events"
 
+orbs:
+  slack: circleci/slack@4.7.0
+
 executors:
   alpine-stable:
     docker:
     - image: alpine:latest
     resource_class: small
 
+commands:
+  slack-on-failure:
+    description: |
+      Uses slack orb to send a notification to slack when a deployment fails.
+    parameters:
+      channel:
+        default: ${SLACK_DEFAULT_CHANNEL}
+        description: |
+          Destination channel for your slack messages.
+        type: string
+    steps:
+      - slack/notify:
+          channel: <<parameters.channel>>
+          custom: |
+            {
+              "text": "",
+              "blocks": [
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": ":alert: *Failure* :alert: \n - Job name: `${CIRCLE_JOB}` \n - Repository name: `${CIRCLE_PROJECT_REPONAME}` \n - Branch `${CIRCLE_BRANCH}` \n - Blame :point_right: `@${CIRCLE_USERNAME}` \n - Build number: #${CIRCLE_BUILD_NUM}"
```

Comment:
Some aesthetics comments 😄 Perhaps :alert: is too noisy? Generally when we have used that in the past, it was more of a "everybody make sure you see this!" use case. Is that what this new feature is intended for? Otherwise perhaps something like ❌  is sufficient.

Also it seem mentioning the user within backticks doesn't actually ping them on slack (from the looks of it within #voiceai-deploys-dev)

Reply:
I agree with :alert: being too noisy. Although this message may be important because it will be sent only when a circleci job fails on production, it is not necessary to highlight that much. I replaced :alert: for ❌. Additionally, I removed backticks since they are worthless. The transformation from GitHub username to Slack usernames should be our next step.